docs/_sidebar.md

```diff
@@ -1,7 +1,10 @@
 - [Overview](home)
 - [Getting Started](quickstart)
 - [Onboarding Public Officers using non-SE Device](onboard-public-officers-using-non-se-machines)
-  - [Log in to TechPass Portal](log-into-techpass-portal-po-non-se)
+- Manage security verification
+  - [Set up security verification for WOG accounts](manage-security-verification-for-wog-account)
+  - [Re set up security verification for WOG account](re-set-up-security-verification-for-wog-account)
+- [Log in to TechPass Portal](log-into-techpass-portal-po-non-se)
```

Comment:
qns ah.. looking at the sidebar item. 
[Log in to TechPass Portal] < seems to cater to everyone . both po and vendors, etc.
(log-into-techpass-portal-po-non-se) < but this md file seems to named for jus non-se po. is that the intention?
if yes, then u might wan to rename sidebar item. if not, rename the md file

Reply:
renamed the file.